examples/get_driver.toit

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2021 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the LICENSE file.
+
+import e_paper.waveshare_2_color_1_54 show *
+import gpio
+import pixel_display show AbstractDriver
```

Comment:
I think we should create `Driver` interface, or something. Feels wrong to use `AbstractDriver` as a user.
(obviously not this PR).